### ISSUE TITLE
bug: false positive for `literal_string_with_formatting_args`  lint

### ISSUE THREAD

**Issue by dj8yfo** [rust-lang/rust-clippy#14363]
### Description

Appears in ci: https://github.com/near/near-workspaces-rs/actions/runs/13703888844/job/38324340265

Silenced with https://github.com/near/near-workspaces-rs/pull/403/commits/2f7789a344e28fdef60fd74dbe139344fd047c1d

### Version

```text

```

**Comment by y21** [comment]
Do you also see this on nightly? This looks somewhat like #13885 which is already fixed (but only on nightly for now). Running `-Dclippy::nursery` in CI is certainly brave though :)

**Comment by dj8yfo** [comment]
@y21 oh, i didn't know a thing about -Dclippy::nursery. Are these some kind of fresh/unstable lints?
